File: src/main/java/com/trade/securities/application/port/in/GetDerivativesTradingInfoUseCase.java
package com.trade.securities.application.port.in;

import com.trade.securities.domain.DerivativesTradingInfo;
import java.util.List;

public interface GetDerivativesTradingInfoUseCase {
    List<DerivativesTradingInfo> getDerivativesTradingInfo(String marketType, String basDd);
}

File: src/main/kotlin/com/trade/securities/application/port/in/GetDerivativesTradingInfoUseCase.kt
package com.trade.securities.application.port.`in`

import com.trade.securities.domain.DerivativesTradingInfo

interface GetDerivativesTradingInfoUseCase {
    fun getDerivativesTradingInfo(marketType: String, basDd: String): List<DerivativesTradingInfo>
}
